cinema/models.py

```diff
@@ -84,16 +88,21 @@
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @property
+    def full_name(self):
+        return {"row": self.row, "seat": self.seat}
+
+    @staticmethod
+    def validate_seat(seat, row, movie_session, error_to_raise):
         for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
+            (row, "row", "rows"),
+            (seat, "seat", "seats_in_row"),
```

Comment:
The `validate_seat` method correctly checks if the seat and row numbers are within the valid range. Ensure that the `error_to_raise` parameter is always passed a valid exception class, such as `ValidationError`, as it is used to raise an error if validation fails.